includes/class-wc-frontend-scripts.php

```diff
@@ -355,7 +355,7 @@ public static function load_scripts() {
 		if ( is_cart() ) {
 			self::enqueue_script( 'wc-cart' );
 		}
-		if ( is_cart() || is_checkout() || is_account_page() ) {
+		if ( is_cart() || is_checkout() || is_account_page() || is_product() ) {
```

Comment:
Why was this included?

Reply:
Removed that. Please [check](https://github.com/woocommerce/woocommerce/pull/23290/commits/826c88a4161548c29d7058fc131c7866801c6a48).